.travis.yml

```diff
@@ -0,0 +1,4 @@
+script:
```

Comment:
It's working without, but [Travis recommends](https://docs.travis-ci.com/user/docker/) specifying the dependency on docker explicitly, so e.g. before this script adding:
```
services:
- docker
```